Dockerfile

```diff
@@ -4,6 +4,6 @@ EXPOSE 9090
 ENV db_hostname dcsa_db
 COPY run-in-container.sh /run.sh
 RUN chmod +x /run.sh
-COPY src/main/resources/application.yaml .
-COPY target/dcsa_jit-*.jar .
+COPY jit-service/src/main/resources/application.yml .
```

Comment:
You should not need to copy application.yml, it should be part of dcsa-jit-service.jar